Library/Homebrew/formula.rb

```diff
@@ -1288,9 +1290,18 @@ def system(cmd, *args)
           end
           wr.close
 
-          while buf = rd.gets
-            log.puts buf
-            puts buf
+          if verbose
+            while buf = rd.gets
+              log.puts buf
+              puts buf
+            end
+          else # travis
+            while buf = rd.gets
+              log.puts buf
+              print "."
```

Comment:
Have you tried this with e.g. `boost`? It seems like it'd produce a vast number of dots.